src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTab.cmp-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AuraDefinitionBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>43.0</apiVersion>
+    <description>Batch Gift Entry to be used in a custom product tab</description>
```

Comment:
Batch Gift Entry Tab component to be used in a custom tab to display gift batches

Reply:
Checking this off with namespace removal PR.